wrangler.toml

```diff
@@ -24,10 +24,14 @@ kv_namespaces = [
 ]
 [env.dev.vars]
 FOUNDRY_ENV = "dev"
+# TODO: swap this to be a bucket/server that's actually for foundry. This is just a placeholder
+FOUNDRY_MFE_HOST = "https://react.microfrontends.app/"
```

Comment:
Now that we have envs, we have to specify all env var values for env

Reply:
Ah, I was under the impression that wrangler would infer that the variable defined in the root environment would exist in the other environments unless overwritten. I see now that that's not the case.